requirements.txt

```diff
@@ -4,7 +4,8 @@ black
 cleo>=0.8.1,<0.9
 cryptography>=3.3.1,<4.0
 dotty_dict>=1.3.0<1.40
-exceptionite>=1.0,<1.1
+# exceptionite>=2.0,<3
+https://github.com/girardinsamuel/exceptionite/archive/refs/heads/2.0-dev.zip
```

Comment:
once exceptionite is officialy release we switch those lines and then the PR should still pass and we can merge !